etc/profile-a-l/evolution.profile

```diff
@@ -71,7 +73,8 @@ shell none
 tracelog
 
 # disable-mnt
-# Add "gpg,gpg2,gpg-agent,pinentry-curses,pinentry-emacs,pinentry-fltk,pinentry-gnome3,pinentry-gtk,pinentry-gtk2,pinentry-gtk-2,pinentry-qt,pinentry-qt4,pinentry-tty,pinentry-x2go,pinentry-kwallet" for gpg 
+# Add "gpg,gpg2,gpg-agent,pinentry-curses,pinentry-emacs,pinentry-fltk,pinentry-gnome3,pinentry-gtk,pinentry-gtk2,pinentry-gtk-2,pinentry-qt,pinentry-qt4,pinentry-tty,pinentry-x2go,pinentry-kwallet" for gpg
+# Add "sh,bash,which,firefox" for hyperlinks 
```

Comment:
The private-bin for firefox is distro-specific, this works only for Arch.

Reply:
xdg-open or something else? I use this on various profiles on Xubuntu with 0.9.64-2,assuming a firefox is running...

Reply:
Since it's distro dependent, I just remove that line?